VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts

```diff
@@ -119,7 +120,9 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
 
     getPrivateConsultationParticipants(): ParticipantListItem[] {
         return this.participantsInConsultation
-            .filter(c => c.hearing_role !== HearingRole.WITNESS && c.hearing_role !== HearingRole.OBSERVER)
+            .filter(
+                c => c.hearing_role !== HearingRole.WITNESS && c.hearing_role !== HearingRole.OBSERVER && c.role !== Role.QuickLinkObserver
```

Comment:
both the hearing role and the role gets mapped in the backend of video web inside of ParticipantRequestMapper.cs, so I think it should be sufficient enough to check the role here, as hearing role itself is a string value and not sure how reliable that can be. Unless we are using hearing role here for some reason

Reply:
This does use role for QuickLinksObserver. We cannot use it for Witness or Observer though. Is that what you were suggesting?

Reply:
yea you are right, I misread the hearing role observer to be quick link hearing role observer, this is fine and I will mark it as resolved